src/components/OrderCard/OrderCard.tsx

```diff
@@ -0,0 +1,27 @@
+import { OrderDto } from 'dtos/Order/OrderDto';
+import { ReactElement } from 'react';
+import { GBPFormat } from 'utilities/GBPFormat';
+
+interface OrdersCardProps {
+  order: OrderDto;
+}
+
+export const OrdersCard = ({ order }: OrdersCardProps): ReactElement => {
+  return (
+    <div className="mt-2 w-full rounded-lg bg-white px-6 pt-6">
+      <div className="w-full border-y-2 border-gray-100">
+        {order.lineItems.map((lineItem) => (
+          <div key={lineItem.product.id} className="py-2">
+            {lineItem.quantity}x {lineItem.product.name}
+          </div>
+        ))}
+      </div>
+      <div className="my-2 flex justify-between pb-4 font-light">
+        <span className="text-xs">
+          {order.createdOn.toString().slice(8, 10)}/{order.createdOn.toString().slice(5, 7)}/{order.createdOn.toString().slice(0, 4)}
```

Comment:
there's a better way to do this - create a utility class that uses either .GetDay, .GetMonth etc, or .useLocaleString - something like that

Reply:
will need to use this for the above date too

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

Reply:
using any kind of class function here has been giving me errors like `Uncaught TypeError: date.getDate is not a function` with the exception of `.toString()`

Reply:
@hazel-davis create a utility component like this - 

export const DateUtility = ({ date }: Props): ReactElement => {
const date = new Date(date). - *and then do whatever here*
}